canarieapi-cron

```diff
@@ -1,2 +1,2 @@
-* * * * * root python -c 'from canarieapi import logparser; logparser.cron_job()' >> /var/log/logparser_cron.log 2>&1
-* * * * * root python -c 'from canarieapi import monitoring; monitoring.cron_job()' >> /var/log/monitoring_cron.log 2>&1
+* * * * * root python3 -c 'from canarieapi import logparser; logparser.cron_job()' >> /var/log/logparser_cron.log 2>&1
+* * * * * root python3 -c 'from canarieapi import monitoring; monitoring.cron_job()' >> /var/log/monitoring_cron.log 2>&1
```

Comment:
For this I think the best way would be for the calling process to do a symlink from `python` to `python3`.
The cron file remains independent of the actual python name.

Reply:
The calling process is the service cron, so I guess that in the dockerfile we could add the symlink after the python3 install.

Reply:
Change is reverted and symlink is added to dockerfile